pf4j/src/main/java/ro/fortsoft/pf4j/PluginStatusProvider.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2012 Decebal Suiu
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this work except in compliance with
+ * the License. You may obtain a copy of the License in the LICENSE file, or at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package ro.fortsoft.pf4j;
+
+import java.util.List;
+
+/**
+ * @author Decebal Suiu
+ * @author Mário Franco
+ */
+public interface PluginStatusProvider {
+
+    /**
+     * Retrieves a list with plugin identifiers that should be only accepted by this manager
+     */
+    public List<String> getEnabledPlugins();
```

Comment:
I only saw that now that you mentioned and I think it's the best option. 
I will clean up the code.